EasyPost.Net/Address.cs

```diff
@@ -138,12 +138,32 @@ public static Address Create(Dictionary<string, object> parameters = null)
                 verifications = (List<string>)parameters["verifications"];
                 parameters.Remove("verifications");
             }
+            else if (parameters.ContainsKey("verify"))
+            {
+                verifications = new List<string>();
+                foreach (bool val in (List<bool>)parameters["verify"])
+                {
+                    verifications.Add(val.ToString());
+                }
+
+                parameters.Remove("verify");
+            }
 
             if (parameters.ContainsKey("strict_verifications"))
             {
                 strictVerifications = (List<string>)parameters["strict_verifications"];
                 parameters.Remove("strict_verifications");
             }
+            else if (parameters.ContainsKey("verify_strict"))
```

Comment:
Looks like C# never handle when there is `verify` and `verify_strict`. Can you add this to the changelog?

Reply:
Resolved in ef038dc01e2831d2938035fa0ff2541909a4dd5f